internal/matomo/matomo.go

```diff
@@ -0,0 +1,119 @@
+package matomo
+
+import (
+	"crypto/rand"
+	"fmt"
+	"math/big"
+	"net/http"
+	"net/url"
+	"os"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/scaleway/scaleway-cli/internal/terminal"
+	"github.com/scaleway/scaleway-sdk-go/logger"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	// full reference API https://developer.matomo.org/api-reference/tracking-api
+	endpoint     = "kindermoumoute.matomo.cloud/matomo.php"
```

Comment:
shouldn't we use an env var ?

Reply:
only CLI developers require to change this endpoint